kiota_authentication_azure/azure_identity_authentication_provider.py

```diff
@@ -15,6 +15,7 @@ def __init__(
         self,
         credentials: Union["TokenCredential", "AsyncTokenCredential"],
         options: Optional[Dict] = None,
+        is_cae_enabled: bool = True,
```

Comment:
inserting a parameter in the middle like that will break people who do not rely on named parameters and simply rely on the order of things.
I believe this new parameter should be at the end.